org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/internal/wizard/workspacecomposite/HelpContextIdProvider.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2015 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Simon Scholz (vogella GmbH) - initial API and implementation and initial documentation
+ *     Sebastian Kuzniarz (Diebold Nixdorf Inc.) - adaptation and customization for workspace composite wizard 
+ */
+
+package org.eclipse.buildship.ui.internal.wizard.workspacecomposite;
+
+/**
+ * Implemented by components for which there is a help context id.
+ */
+public interface HelpContextIdProvider {
```

Comment:
❌  Unnecessary duplicate class.

Reply:
Maybe then we should move this class to `org.eclipse.buildship.ui.internal`, so the project and the workspace composite wizards both use the same class without causing confusion due to the file management